README.rst

```diff
@@ -65,7 +65,6 @@ Example
 .. code-block:: python
 
     import dimod
-    import hybrid
```

Comment:
We pretty much need `hybrid` in this example. :)

Reply:
oops